src/Spells/Spells.js

```diff
@@ -1,22 +1,33 @@
 import './Spells.css';
 import Card from '../Card/Card.js';
+import PropTypes from 'prop-types';
+
+export default function Spells({spells, handleKnown, handleDelete}) {
 
-export default function Spells({spells}) {
   const spellCards = spells.map((spell) => (
     <Card
-      key={spell.index}
-      id={spell.index}
+      key={spell.name}
```

Comment:
did you do this because of html elements? that's part of why I did it on mine. my pokemon were generating as numbers which wasn't very useful to the user

Reply:
No, it was actually cuz the api data didn't have any numbers to grab and use.